features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Documents/Details.jsx

```diff
@@ -38,6 +34,11 @@ import MenuItem from '@material-ui/core/MenuItem';
 import Input from '@material-ui/core/Input';
 import { Link } from 'react-router-dom';
 import { doRedirectToLogin } from 'AppComponents/Shared/RedirectToLogin';
+import Api from 'AppData/api';
+import APIProduct from 'AppData/APIProduct';
+import PageContainer from 'AppComponents/Base/container/';
+import ResourceNotFound from 'AppComponents/Base/Errors/ResourceNotFound';
+import PageNavigation from '../../APIsNavigation';
```

Comment:
Shall we use `AppComponents` alias here

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/6759/commits/4f501bf1f995ac5a20a84923f720b71550950f0a